src/plugin.js

```diff
@@ -161,20 +163,24 @@ const onPlayerReady = (player, options) => {
     let details = '';
     let error = player.error();
     let content = document.createElement('div');
+    let flashMessage = '';
 
     // In the rare case when `error()` does not return an error object,
     // defensively escape the handler function.
     if (!error) {
       return;
     }
-
+    if (error.code === 4 && !FlashObj.isSupported()) {
+      flashMessage = player.localize(' * If you are using an older browser' +
+      ' please try upgrading or installing Flash.');
+    }
     error = videojs.mergeOptions(error, options.errors[error.code || 0]);
-
     if (error.message) {
       details = `<div class="vjs-errors-details">${player.localize('Technical details')}
         : <div class="vjs-errors-message">${player.localize(error.message)}</div>
         </div>`;
     }
+    details += `<span class="vjs-errors-flashmessage">${flashMessage}</span>`;
```

Comment:
Sorry, I meant that lines 166 and 173-176 should be removed and here have:

```js
if (error.code === 4 && !FlashObj.isSupported()) {
  const flashMessage = player.localize('...');

  details += `<span class="vjs-errors-flashmessage">${flashMessage}</span>`;
}
```
That would keep the changes localized to a single `if` block instead of 3 places.